src/components/EtherAddress.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+import { isValidAddress } from 'ethereumjs-util';
+
+// Accepts checksummed addresses too
+const EtherAddress = address => isValidAddress(address);
```

Comment:
This does not return Html (but boolean) so it is not a React component but a regular function.